File: backEnd/ishtar-spring/src/main/java/com/ninos/dto/PurchaseRequest.java
package com.ninos.dto;

import com.ninos.model.form.Address;
import com.ninos.model.form.Customer;
import com.ninos.model.form.Item;
import com.ninos.model.form.Request;

import lombok.Data;


import java.util.ArrayList;

import java.util.List;

@Data
public class PurchaseRequest {

    private Customer customer;
    private Request requestOrder;
    private List<Item> items = new ArrayList<>();
    private Address fromAddress;
    private Address toAddress;

}

File: frontEnd/ishtar-angular/src/app/model/form/purchase-request.ts
import { Item } from './item';
import { Address } from './address';
import { Customer } from './customer';
import { Request } from './request'

export class PurchaseRequest {


  customer?: Customer;
  fromAddress?: Address;
  toAddress?: Address;
  requestOrder?: Request;
  items?: Item[];




}
